charts/flyte-core/templates/console/deployment.yaml

```diff
@@ -30,6 +30,13 @@ spec:
             name: flyte-console-config
         ports:
         - containerPort: 8080
+        {{- if .Values.flyteconsole.ga.enabled }}
+        env:
+          - name: ENABLE_GA
+            value: {{ .Values.flyteconsole.ga.enabled }}
```

Comment:
Since this is already in if condition with the same value,  do we need to maintain this variable .

Reply:
Why can't the app just use the tracking id if it's not empty that means ga tracking is enabled.

I find another env variable redundant here

Reply:
Check this https://github.com/flyteorg/flyteconsole/pull/244

Reply:
great thanks for additional background.The app could have used one but sure since it already implemented using two variables then we can go ahead with this.